src/Integration.Vsix/SonarLintTagger/TextBufferIssueTracker.cs

```diff
@@ -184,29 +176,25 @@ private void SafeOnBufferChange(object sender, TextContentChangedEventArgs e)
                 // in the new text buffer.
                 currentSnapshot = e.After;
 
-                var newMarkers = TranslateMarkerSpans();
-
-                SnapToNewSnapshot(newMarkers);
+                var newMarkers = TranslateSpans(Factory.CurrentSnapshot.IssueMarkers, currentSnapshot);
+                UpdateIssues(newMarkers);
```

Comment:
Minor refactoring so there is only one "translate" method and only one method that creates a new snapshot.